### PR TITLE
less TICI when not needed

### PR DESCRIPTION
**Pull request by pd0wm**
Needs process replay regen due to different DM and planner config values.

### CODE REVIEW

File: selfdrive/hardware/__init__.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from selfdrive.hardware.pc.hardware import Pc
 
 TICI = os.path.isfile('/TICI')
+AGNOS = TICI
```
Comment (pd0wm) [inline selfdrive/hardware/__init__.py:9]: `/AGNOS` will be added in agnos 5

File: launch_chffrplus.sh [hunk 2]
```diff
@@ -77,9 +77,7 @@ function launch {
   export PYTHONPATH="$PWD:$PWD/pyextra"
 
   # hardware specific init
-  if [ -f /TICI ]; then
-    tici_init
-  fi
+  tici_init
```
Comment (robbederks) [inline launch_chffrplus.sh:80]: `agnos_init` ?

File: selfdrive/boardd/boardd.cc [hunk 3]
```diff
@@ -405,17 +405,12 @@ void send_peripheral_state(PubMaster *pm, Panda *panda) {
...
-    ps.setVoltage(std::atoi(util::read_file("/sys/class/hwmon/hwmon1/in1_input").c_str()));
-    ps.setCurrent(std::atoi(util::read_file("/sys/class/hwmon/hwmon1/curr1_input").c_str()));
-    read_time = millis_since_boot() - read_time;
-    if (read_time > 50) {
-      LOGW("reading hwmon took %lfms", read_time);
-    }
-  } else {
-    ps.setVoltage(pandaState.voltage_pkt);
-    ps.setCurrent(pandaState.current_pkt);
+  double read_time = millis_since_boot();
```
Comment (robbederks) [inline selfdrive/boardd/boardd.cc:408]: doesn't this break on PC?

Comment (pd0wm) [reply selfdrive/boardd/boardd.cc:408]: ah, good catch

File: launch_chffrplus.sh (hunk 2 above)
Comment (crwusiz) [reply launch_chffrplus.sh:80]: line 11

function tici_init {

-> function agnos_init {

Comment (crwusiz) [reply launch_chffrplus.sh:80]: line 11

function tici_init {

-> function agnos_init {

Comment (adeebshihadeh) [reply launch_chffrplus.sh:80]: no reason this shouldn't work on PC

File: selfdrive/hardware/base.h [hunk 4]
```diff
@@ -10,6 +11,10 @@ class HardwareNone {
   static constexpr float MIN_VOLUME = 0.2;
 
   static std::string get_os_version() { return ""; }
+  static std::string get_name() { return ""; };
+  static cereal::InitData::DeviceType get_device_type() {return cereal::InitData::DeviceType::UNKNOWN; };
+  static int get_voltage() {return 0; };
+  static int get_current() {return 0; };
```
Comment (adeebshihadeh) [inline selfdrive/hardware/base.h:17]:
```suggestion
  static cereal::InitData::DeviceType get_device_type() { return cereal::InitData::DeviceType::UNKNOWN; };
  static int get_voltage() { return 0; };
  static int get_current() { return 0; };
```

File: selfdrive/hardware/hw.h [hunk 5]
```diff
@@ -10,8 +10,11 @@
 class HardwarePC : public HardwareNone {
 public:
   static std::string get_os_version() { return "openpilot for PC"; }
+  static std::string get_name() { return "pc"; };
+  static cereal::InitData::DeviceType get_device_type() {return cereal::InitData::DeviceType::PC; };
```
Comment (adeebshihadeh) [inline selfdrive/hardware/hw.h:14]:
```suggestion
  static cereal::InitData::DeviceType get_device_type() { return cereal::InitData::DeviceType::PC; };
```

File: selfdrive/hardware/tici/hardware.h [hunk 6]
```diff
@@ -12,9 +12,15 @@ class HardwareTici : public HardwareNone {
...
   static constexpr float MIN_VOLUME = 0.2;
   static bool TICI() { return true; }
+  static bool AGNOS() { return true; }
   static std::string get_os_version() {
     return "AGNOS " + util::read_file("/VERSION");
   };
+  static std::string get_name() { return "tici"; };
+  static cereal::InitData::DeviceType get_device_type() {return cereal::InitData::DeviceType::TICI; };
+  static int get_voltage() {return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/in1_input").c_str()); };
+  static int get_current() {return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/curr1_input").c_str()); };
```
Comment (adeebshihadeh) [inline selfdrive/hardware/tici/hardware.h:22]:
```suggestion
  static cereal::InitData::DeviceType get_device_type() { return cereal::InitData::DeviceType::TICI; };
  static int get_voltage() { return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/in1_input").c_str()); };
  static int get_current() { return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/curr1_input").c_str()); };
```

File: selfdrive/loggerd/uploader.py [hunk 7]
```diff
@@ -15,7 +15,7 @@
 from common.api import Api
 from common.params import Params
 from common.realtime import set_core_affinity
-from selfdrive.hardware import TICI
+from selfdrive.hardware import AGNOS
```
Comment (pd0wm) [inline selfdrive/loggerd/uploader.py:18]:
```suggestion
from selfdrive.hardware import TICI
```

File: selfdrive/loggerd/uploader.py [hunk 8]
```diff
@@ -239,7 +239,7 @@ def uploader_fn(exit_event):
     cloudlog.info("uploader missing dongle_id")
     raise Exception("uploader can't start without dongle id")
 
-  if TICI and not Path("/data/media").is_mount():
+  if AGNOS and not Path("/data/media").is_mount():
```
Comment (pd0wm) [inline selfdrive/loggerd/uploader.py:242]:
```suggestion
  if TICI and not Path("/data/media").is_mount():
```

File: selfdrive/thermald/thermald.py [hunk 9]
```diff
@@ -286,7 +286,7 @@ def thermald_thread(end_event, hw_queue):
     set_offroad_alert_if_changed("Offroad_TemperatureTooHigh", (not onroad_conditions["device_temp_good"]))
 
     # TODO: this should move to TICI.initialize_hardware, but we currently can't import params there
-    if TICI:
+    if AGNOS:
```
Comment (pd0wm) [inline selfdrive/thermald/thermald.py:289]:
```suggestion
    if TICI:
```

**Pull request merged**